components/director/internal/domain/scenarioassignment/service_test.go

```diff
@@ -354,7 +356,7 @@ func TestService_DeleteManyForSameSelector(t *testing.T) {
 		// GIVEN
 		mockRepo := &automock.Repository{}
 		mockRepo.On("DeleteForSelector", ctx, tenantID, selector).Return(nil).Once()
-		mockEngineSvc := &automock.AssignmentEngineService{}
+		mockEngineSvc := &automock.AssignmentEngine{}
```

Comment:
What is the best name:
`mockEngine` or `mockEngineSvc`. I would go with `mockEnginge.